pylinac/winston_lutz.py

```diff
@@ -366,6 +366,10 @@ def __init__(self, file):
         self.check_inversion()
         # Flip the image upside down... makes it more intuitive later on.
         self.flipud()
+        # The first pixel line on the left has a glitch on Elekta iview EPID
```

Comment:
The datasets already provided contains this glitch.

I am also able to confirm that this glitch is common for Elektas (4 hospitals, sample of ~ 15 linacs).   It doesn't appears on all of them though­. It affects the field_cax position by about 0.5 mm (for a 10x10 field). Clipping may not be only option, but it is a quick and safe one. An elekta EPID is 1024x1024 pixels.... 1023x1024 doesn't really affect the covered area. Feel free to use  another fix if you prefer.